.circleci/config.yml

```diff
@@ -23,13 +23,14 @@ jobs:
           command: |
             python3 -m venv venv
             . venv/bin/activate
+            bash scripts/install_gradio.sh
```

Comment:
So that gradio is installed in circleci worker. We were doing this implicitly by having the egg-info in version control.